### ISSUE TITLE
Identidades Secretas

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments-new#165]
### Post de Referência:

https://www.vish.blog.br/imagens/identidades-secretas/

**Comment by brunoaugusto** [comment]
Eu meio que fiquei ofendido pelo artista considerar que o Link seria o Peter Pan :triumph:

**Comment by maryon-telvanni** [comment]
Alguns eu nem sei quem são, mas acho que nenhum faz tanto sentido quanto esse:



![null](<https://i.imgur.com/OtvEMGK.png>)

**Comment by najarin-the-loremaster** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/identidades-secretas/#comment_1187704647>):

> Alguns eu nem sei quem são, mas acho que nenhum faz tanto sentido quanto esse:[![null](<https://camo.githubusercontent.com/6539b97f0c8a8178fe215edba9b14e98497250acf9c7c9d6fc307b4fae2ee950/68747470733a2f2f692e696d6775722e636f6d2f4f7476454d474b2e706e67>)](<https://camo.githubusercontent.com/6539b97f0c8a8178fe215edba9b14e98497250acf9c7c9d6fc307b4fae2ee950/68747470733a2f2f692e696d6775722e636f6d2f4f7476454d474b2e706e67>)

Concordo. Existem até pelo menos duas teorias bastante sólidas sobre isso:



Em uma delas, Samurai Jack se passa num futuro onde Aku teria derrotado as Meninas e destruído Townsville e o "Jack" seria o Professor Utônio buscando vingança; em outra, o Elemento X seria uma substância mutagênica presente na natureza da qual tanto o Aku quanto suas Sete Filhas se originaram, vivendo em confronto eterno com "Jack" que tem de se adaptar ao longo dos anos e assume a personalidade de cientista, disfarce perfeito para criar as Meninas como seu legado.
